paparazzi/src/main/java/app/cash/paparazzi/accessibility/AccessibilityRenderExtension.kt

```diff
@@ -381,6 +413,17 @@ public class AccessibilityRenderExtension : RenderExtension {
     private const val LIVE_REGION_POLITE_LABEL = "polite"
     private const val EDITABLE_LABEL = "<editable>"
     private const val IN_LIST_LABEL = "<in-list>"
+
+    data class SemanticsNodeTraversalEntry(
+      val traversalIndex: Float?,
```

Comment:
traversalIndex should default to 0 for all elements. We might want to consider this instead of having a nullable Float.
Cash uses `traversalIndex = -1` to force elements to be read first. We should ensure this works as expected in the test

Reply:
I'll try this out and update the test case to make sure negative traversalIndices are supported properly!

Reply:
Just updated this PR to include the default 0 value. Thanks for pointing this out, it actually was a bug before. The order of the focusable elements in the modal bottom sheet changed as a result of implementing this behaviour correctly.